pkg/controller/rbac/provider/binding/watch.go

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2020 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package binding
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/util/workqueue"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+
+	"github.com/crossplane/crossplane/apis/pkg/v1alpha1"
+)
+
+type adder interface {
+	Add(item interface{})
+}
+
+// EnqueueRequestForRevision enqueues a reconcile request for all provider
+// reference owners of a service account.
+type EnqueueRequestForRevision struct{}
+
+// Create enqueues a reconcile request for all provider reference owners of the
+// supplied object if it is a service account.
+func (e *EnqueueRequestForRevision) Create(evt event.CreateEvent, q workqueue.RateLimitingInterface) {
+	addRevision(evt.Object, q)
+}
+
+// Update enqueues a reconcile request for all provider reference owners of the
+// supplied object if it is a service account.
+func (e *EnqueueRequestForRevision) Update(evt event.UpdateEvent, q workqueue.RateLimitingInterface) {
+	addRevision(evt.ObjectOld, q)
+	addRevision(evt.ObjectNew, q)
+}
+
+// Delete enqueues a reconcile request for all provider reference owners of the
+// supplied object if it is a service account.
+func (e *EnqueueRequestForRevision) Delete(evt event.DeleteEvent, q workqueue.RateLimitingInterface) {
+	addRevision(evt.Object, q)
+}
+
+// Generic enqueues a reconcile request for all provider reference owners of the
+// supplied object if it is a service account.
+func (e *EnqueueRequestForRevision) Generic(evt event.GenericEvent, q workqueue.RateLimitingInterface) {
+	addRevision(evt.Object, q)
+}
+
+func addRevision(obj runtime.Object, queue adder) {
+	sa, ok := obj.(*corev1.ServiceAccount)
+	if !ok {
+		return
+	}
+	for _, ref := range sa.GetOwnerReferences() {
+		if ref.APIVersion != v1alpha1.ProviderRevisionGroupVersionKind.GroupVersion().String() {
+			return
```

Comment:
```suggestion
			continue
```
Is the assumption that there will ever be only one ownerref feasible?

Reply:
There will most likely only be one owner ref and it will be a ProviderRevision, but my intent here was to assess all of them just in case so I agree `continue` would be better.